javaslang/src/main/java/javaslang/collection/IndexedSeq.java

```diff
@@ -220,19 +221,30 @@ default int lastIndexWhere(Predicate<? super T> predicate, int end) {
 
     @Override
     default Iterator<T> reverseIterator() {
-        return new AbstractIterator<T>() {
-            private int i = IndexedSeq.this.length();
+        return new AbstractIterator<T>() { /*@formatter:off*/
+            final ListIterator<T> delegate = IndexedSeq.this.listIterator(IndexedSeq.this.length());
+            @Override public boolean hasNext() { return delegate.hasPrevious(); }
+            @Override public T getNext()       { return delegate.previous(); }
```

Comment:
Please remove the `formatter:off`. I will remove all these occurrences from our code base. We use a formatter to unify the code. Either we use a formatter or we do not use it. But switching code parts on and off is like hell.

It reminds me of checkstyle tweaks: "We want to have line length X. But in these cases we want to have line length X+Y".

Don't be too obsessed with formatting. We have a formatter, so just apply it and forget about formatting. I think you are a mirror of myself. I also tend to get carried away and rambling on.